javascript-conditionals/main.js

```diff
@@ -0,0 +1,126 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+const number = 3;
+function isUnderFive(number) {
+  return number < 5;
+}
+console.log('Is under five: ', isUnderFive(number));
+
+function isEven(number) {
+  if (number % 2) {
+    return false;
+  } else {
+    return true;
+  }
+}
+console.log('Is even:', isEven(number));
+
+const string = 'Hello';
+function startsWithJ(string) {
+  if (string.charAt(0) === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log('Does it start with J:', startsWithJ(string));
+
+const person = {
+  name: 'Andrew Ortiz',
+  Age: 27,
+};
+function isOldEnoughToDrink(person) {
+  if (person >= 21) {
+    return false;
+  } else {
+    return true;
+  }
+}
+console.log('old enough to drink:', isOldEnoughToDrink(person));
+
+function isOldEnoughToDrive(person) {
+  if (person >= 16) {
+    return false;
+  } else {
+    return true;
+  }
+}
```

Comment:
If the person is 16 or older, that means they are old enough to drive, so you should be `return true`